? stf/src/all.src
? stf/src/appendsrc.sh
? stf/src/icecube.daq.stf.autogen.cf
? stf/src/icecube.daq.stf.autogen.deps
? stf/src/nlines.sh
Index: stf/private/stf-apps/atwd_led_nopmt.c
===================================================================
RCS file: /home/icecube/cvsroot/stf/private/stf-apps/atwd_led_nopmt.c,v
retrieving revision 1.1.2.1
diff -u -r1.1.2.1 atwd_led_nopmt.c
--- stf/private/stf-apps/atwd_led_nopmt.c	4 Mar 2005 09:25:46 -0000	1.1.2.1
+++ stf/private/stf-apps/atwd_led_nopmt.c	13 Apr 2005 23:38:44 -0000
@@ -181,5 +181,5 @@
 
    free(buffer1);
    return       
-     ((LED_dac==0 && *real_LED_voltage>=16 && *real_LED_voltage<=17) || (LED_dac==1023 && *real_LED_voltage>=0 && *real_LED_voltage<=3));
+     ((LED_dac==0 && *real_LED_voltage>=16 && *real_LED_voltage<=20) || (LED_dac==1023 && *real_LED_voltage>=0 && *real_LED_voltage<=3));
 }
